pyproject.toml

```diff
@@ -20,6 +20,8 @@ python-dotenv = "^1.0.0"
 requests = "^2.31.0"
 langchain-openai = "^0.0.3"
 llama-index = "^0.9.40"
+guardrails-ai = "^0.4.0"
+torch = "^2.2.0"
```

Comment:
I feel this will be very heavy for people who don't want this particular eval.
Maybe we do some lazy loading or have a different way of getting this dependencies.